pilot/docker/envoy_policy.yaml.tmpl

```diff
@@ -0,0 +1,126 @@
+admin:
+  access_log_path: /dev/stdout
+  address:
+    socket_address:
+      address: 127.0.0.1
+      port_value: 15000
+static_resources:
+  clusters:
+  - circuit_breakers:
+      thresholds:
+      - max_connections: 100000
+        max_pending_requests: 100000
+        max_requests: 100000
+        max_retries: 3
+    connect_timeout: 1.000s
+    hosts:
+    - socket_address:
+        address: 127.0.0.1
+        port_value: 9091
+    http2_protocol_options: {}
+    name: in.9091
+  - connect_timeout: 1.000s
+    hosts:
+    - socket_address:
+        address: zipkin
+        port_value: 9411
+    name: zipkin
+    type: STRICT_DNS
+  - circuit_breakers:
+      thresholds:
+      - max_connections: 100000
+        max_pending_requests: 100000
+        max_requests: 100000
+        max_retries: 3
+    connect_timeout: 1.000s
+    hosts:
+    - socket_address:
+        address: istio-telemetry
+        port_value: 15004
+    http2_protocol_options: {}
+    name: mixer_report_server
+{{- if .ControlPlaneAuth }}
+    tls_context:
+      common_tls_context:
+        tls_certificates:
+        - certificate_chain:
+            filename: /etc/certs/cert-chain.pem
+          private_key:
+            filename: /etc/certs/key.pem
+        validation_context:
+          trusted_ca:
+            filename: /etc/certs/root-cert.pem
+          verify_subject_alt_name:
+          - spiffe://cluster.local/ns/{{ .PodNamespace }}/sa/istio-mixer-service-account
+{{- end }}
+    type: STRICT_DNS
+  listeners:
+  - address:
+      socket_address:
+        address: 0.0.0.0
+        port_value: 15004
+    filter_chains:
+    - filters:
+      - config:
+          access_log:
+          - config:
+              path: /dev/stdout
+            name: envoy.file_access_log
+          codec_type: HTTP2
+          generate_request_id: true
+          http_filters:
+          - config:
+              default_destination_service: istio-policy.{{ .PodNamespace }}.svc.cluster.local
+              service_configs:
+                istio-policy.{{ .PodNamespace }}.svc.cluster.local:
+                  disable_check_calls: true
+                  mixer_attributes:
+                    attributes:
+                      destination.service:
+                        string_value: istio-policy.{{ .PodNamespace }}.svc.cluster.local
+                      destination.uid:
+                        string_value: kubernetes://{{ .PodName }}.{{ .PodNamespace }}
+              transport:
+                check_cluster: mixer_check_server
```

Comment:
perhaps this was discussed before, but why not `in.9090` ?

Reply:
It's not implemented since check is explicitly disabled. There is in fact no mixer_check_server cluster defined.